runtime-testsuite/test/org/antlr/v4/test/runtime/RuntimeTestDescriptor.java

```diff
@@ -1,65 +1,84 @@
-/*
```

Comment:
removing copyright?

Reply:
It's not intentially. I can restore if it matters.

Reply:
Might as well keep the copyright on everything.